Automattic/wp-calypso
# Editor Deprecation: Remove deprecation group state

#### Changes proposed in this Pull Request

Now that the editor is deprecated for all we can remove the logic that checks if the current user is in the deprecation group, as well as the dialog we were displaying in the Calypso editor to encourage people to opt into the block editor.

#### Testing instructions

* Apply D49622-code and sandbox the API
* Run `delete_user_attribute( YOUR_USER_ID, 'editor_deprecation_group')`  in wpsh
* With this PR check that you can't load the Calypso editor, regardless of the editor preference setting for your site
* Also check that you don't encounter any errors either in Calypso or `wp-admin`, when creating and editing posts
